openerp/openupgrade/doc/source/xmlids.rst

```diff
@@ -28,11 +28,11 @@ More interesting are
 
     * res.groups
       If a res groups has moved module (example: hr_manager moved from module
-      hr to base in OpenERP 6), implement this change in the pre-upgrade script
+      hr to base in Odoo 6), implement this change in the pre-upgrade script
       for the module from which it moved.
 
-    * res.roles (migrating from OpenERP 5 to 6 only)
-      These have been migrated to groups in OpenERP6. Find out which group has
+    * res.roles (migrating from Odoo 5 to 6 only)
+      These have been migrated to groups in Odoo 6. Find out which group has
```

Comment:
Well, the product was called OpenERP in that versions, but it's not so important.